Eatery/Controllers/Eateries/ImageParallaxScrollViewController.swift

```diff
@@ -17,7 +17,8 @@ class ImageParallaxScrollViewController: UIViewController {
     private let navigationBar = UINavigationBar()
     private let navigationBarBackground = UIView()
 
-    private let scrollView = UIScrollView()
+    internal let scrollView = UIScrollView()
+    internal var scrollOffset: CGFloat?
```

Comment:
Variables are `internal` by default
```suggestion
    let scrollView = UIScrollView()
    var scrollOffset: CGFloat?
```